gpkit/nomial_data.py

```diff
@@ -107,7 +107,7 @@ def diff(self, var):
     def __eq__(self, other):
         """Equality test"""
         if not all(hasattr(other, a) for a in ("exps", "cs", "units")):
-            return NotImplemented
+            return False
```

Comment:
This was a nice idea @whoburg but doesn't work: turns out that 0 is both `not equal` and `not not equal` to a Signomial, so passing it control is a bad idea. :-/